third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -5197,12 +5209,15 @@ void OnNewWindow1(v8::Isolate* isolate, LocalFrame* localFrame) {
                       ReadFromRecordingDirectory);
   SetFunctionProperty(isolate, args, "getRecordingFilePath",
                       GetRecordingFilePath);
-  SetFunctionProperty(isolate, args, "getPersistentId", GetPersistentId);
-  SetFunctionProperty(isolate, args, "checkPersistentId", CheckPersistentId);
+  SetFunctionProperty(isolate, args, "getPersistentId", fromJsGetPersistentId);
+  SetFunctionProperty(isolate, args, "checkRecordReplayObjectId", CheckRecordReplayObjectId);
 }
 
 void SetupRecordReplayCommands(v8::Isolate* isolate, LocalFrame* localFrame) {
-  V8RecordReplaySetAPIObjectIdCallback(GetAPIObjectIdCallback);
+  // TODO: [RUN-2812] Fix this.
+  // There is some logic in our assertion code that depends on this not
+  // to return ids of plain objects.
```

Comment:
Comments like this should be more specific.  What logic is this referring to?  Having a comment on that logic would be good as well, I didn't see any changes related to this in the other PR.